docs/docs/components/lifecycle-operator/keptn-apps.md

```diff
@@ -67,17 +67,17 @@ for the `KeptnApp` resource itself:
   are used to automatically generate `KeptnApp` resources
   that contain the identifications required
   to run the Keptn observability features.
-* You must manually add the annotations described in
-  [Annotations to KeptnApp](../../guides/tasks.md#annotations-to-keptnapp)
+* You must create a `KeptnAppContext` resource
   to the basic `KeptnApp` manifest to define
-  the evaluations and tasks you want to run pre/post-deployment.
+  the evaluations and tasks you want to run pre-/post-deployment.
```

Comment:
is "create a ... resource to the basic KeptnApp manifest..." the proper phraseology?  It sounds strange to me but maybe it is phraseology I should be using elsewhere?

Reply:
typo fixed :)